arviz/rcparams.py

```diff
@@ -93,11 +102,64 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
+def iterable_validator(validate_scalar, length=None):
+    """Validate value is an iterable datatype."""
+    # no blank lines allowed after function docstring by pydocstyle,
+    # but black requires white line before function
+
+    def validate_iterable(value):
+        if isinstance(value, str):
+            try:
+                val = [validate_scalar(v.strip("([ ])")) for v in value.split(",") if v.strip()]
+                if length is not None:
+                    if len(val) == length:
+                        return val
+                    else:
+                        raise ValueError("Iterable must be of length: {}".format(length))
+                else:
+                    return val
+            except Exception:
+                raise
```

Comment:
@OriolAbril, should I add a ```allow_stringlist``` argument to handle this exception like ```_listify_validator``` or do we not need it right now (we may need it in future) and find another way of handling this exception?

Reply:
No need for `allow_stringlist`. I don't think the case it covers (interpreting `"rgbky"` as a list `["r", "g", "b", "k", "y"]`) will be used in ArviZ